.github/workflows/bundle_with_dakota_macos.yml

```diff
@@ -172,14 +180,23 @@ jobs:
 
         echo "libgccpath=$libgccpath"
         echo "libdakotapath=$libdakotapath"
-
-        DYLD_LIBRARY_PATH="$DYLD_LIBRARY_PATH:$libgccpath:libdakotapath"
+        
+        # This may seem like nonsense but is an intentional
+        # workaround MacOS (flaky) security features that result
+        # in delocate suddenly finding the search paths to be empty
+        # ref issue: https://github.com/pypa/cibuildwheel/issues/816
```

Comment:
This seemed a bit lengthy, can we consider something more to the point like;

```diff
+         # Avoid MacOS security features prohibiting setting DYLD_LIBRARY_PATH
+         # ref issue: https://github.com/pypa/cibuildwheel/issues/816
```

Also, delocate is used 30 lines further down.

Reply:
Moved to just above delocate